.github/workflows/run-benchmarks.yml

```diff
@@ -81,6 +85,21 @@ jobs:
         # Then replace the {0} placeholder with the browser, and the {1} placeholder with the buildType
         run: ${{ format(fromJson(env.COMMANDS)[matrix.pageType], matrix.browser, matrix.buildType) }}
 
+      - name: Send benchmark results to Sentry
+        if: ${{ vars.SENTRY_DSN_PERFORMANCE }}
+        env:
+          SENTRY_DSN_PERFORMANCE: ${{ vars.SENTRY_DSN_PERFORMANCE }}
+          PR_NUMBER: ${{ github.event.pull_request.number || 'none' }}
+          HEAD_COMMIT_HASH: ${{ github.event.pull_request.head.sha || github.sha }}
+          GITHUB_JOB: ${{ github.job }}
+          GITHUB_REF_NAME: ${{ github.head_ref || github.ref_name }}
```

Comment:
Some of these env vars are repeated.  A `step` will inherit env vars from a `job`.

Reply:
OhhhOhhhOhhh good catch 1c23dfa60cc92dbb15dbc52cf8304c8cf6b44bfc